cfgov/unprocessed/css/pages/ask.less

```diff
@@ -63,34 +65,113 @@
         }
     }
 
-    
-}
-
-/* Results pages */
+    .ask-categories,
+    .ask-search {
+        padding: 0 unit( @grid_gutter-width / 2 / @base-font-size-px, em );
 
-.ask-category-page {
-    h1 .cf-icon {
-        color: @green;
+        .respond-to-range( @bp-med-min, @bp-med-max {
+            padding: 0;
+        } );
     }
 
-    .lead-paragraph {
-        max-width: 600px;
+    .ask-search {
+        .respond-to-max( @bp-xs-max {
+            padding: 0;
+        } );
     }
+
+    
 }
 
+/* Results pages */
+
 .ask-cfpb-page {
     .search-results {
-        max-width: 600px;    
+        max-width: @readable-width;    
     }
 
     .question_summary {
         border-bottom: 1px solid @block__border-bottom;
     }
 
+    .o-search-bar {
```

Comment:
We already have a search molecule, but I'm not sure if you could reuse the existing one.